### PR TITLE
Update WP2 main contacts & move to library 1.3.2

### PR DESCRIPTION
**Pull request by orviz**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -133,17 +133,41 @@ pipeline {
...
-                    'mariojmdavid'
-                )
+            parallel {
+                stage('Notify DEEP') {
+                        steps {
+                                JiraIssueNotification(
+                                    'DEEP',
+                                    'DPM',
+                                    '10204',
+                                    "[preview-testbed] New orchent version ${env.BRANCH_NAME} available",
```
Comment (alberto-brigandi) [inline Jenkinsfile]: Shouldn't it be orchestrator and not orchent?

File: Jenkinsfile [hunk 2]
```diff
@@ -133,17 +133,41 @@ pipeline {
...
+                                    "Check new artifacts at:\n\t- Docker image: [${dockerhub_image_id}|https://hub.docker.com/r/${dockerhub_repo}/tags/]",
+                                    ['wp3', 'preview-testbed', "orchent-${env.BRANCH_NAME}"],
```
Comment (alberto-brigandi) [inline Jenkinsfile]: Shouldn't it be orchestrator and not orchent?

File: Jenkinsfile [hunk 3]
```diff
@@ -133,17 +133,41 @@ pipeline {
...
+                                     'ignacioheredia']                                )              
+                        }
+                }
+                stage('Notify XDC') {
+                        steps {
+                                JiraIssueNotification(
+                                    'XDC',
+                                    'XDM',
+                                    '10100',
+                                    "[preview-testbed] New orchent version ${env.BRANCH_NAME} available",
```
Comment (alberto-brigandi) [inline Jenkinsfile]: Shouldn't it be orchestrator and not orchent?

File: Jenkinsfile [hunk 4]
```diff
@@ -133,17 +133,41 @@ pipeline {
...
+                                    "Check new artifacts at:\n\t- Docker image: [${dockerhub_image_id}|https://hub.docker.com/r/${dockerhub_repo}/tags/]",
+                                    ['WP3', 't3.2', 'preview-testbed', "orchent-${env.BRANCH_NAME}"],
```
Comment (alberto-brigandi) [inline Jenkinsfile]: Shouldn't it be orchestrator and not orchent?

File: Jenkinsfile (hunk 2 above)
Comment (caifti) [reply Jenkinsfile]: fixed

File: Jenkinsfile (hunk 1 above)
Comment (caifti) [reply Jenkinsfile]: fixed. 
Sorry!

File: Jenkinsfile (hunk 4 above)
Comment (caifti) [reply Jenkinsfile]: fixed

File: Jenkinsfile (hunk 3 above)
Comment (caifti) [reply Jenkinsfile]: fixed

**Comment by caifti** [comment]
Hi,

solved... sorry for the error!

**Comment by alberto-brigandi** [comment]
Jenkins test this

**Comment by alberto-brigandi** [comment]
test this please

**Comment by orviz** [comment]
@alberto-brigandi it is important to merge this change asap. Could you check why unit test is failing? Thanks

**Comment by alberto-brigandi** [comment]
Hi @orviz,
the pipeline is failing during the bootstrap phase, not during the actual execution of the unit test, because of the error `Could not find or load main class org.apache.maven.surefire.booter.ForkedBooter`

It never happened to me, but looking on google it seems like it might be a [JVM bug](https://stackoverflow.com/questions/53010200/maven-surefire-could-not-find-forkedbooter-class). What is the version of the JVM installed on the jenkins' nodes?

**Comment by orviz** [comment]
Hi @alberto-brigandi somehow it seems to be solved with the deployment of the new Jenkins instance. Now only the License Compliance check is failing, could you please check this?

**Comment by orviz** [comment]
@alberto-brigandi I saw you already approved and since the test is now passing, let merge it

**Pull request merged**